frontend/src/bundles/studio/components/timeline/timeline.tsx

```diff
@@ -136,6 +142,16 @@ const Timeline: React.FC<Properties> = ({ initialRange, initialItems }) => {
         });
     }, []);
 
+    const updatedScripts = useAppSelector(({ studio }) => studio.scripts);
+    useEffect(() => {
+        setItems((previous) =>
+            setItemsSpan({
+                ...previous,
+                script: getScriptItems(updatedScripts),
+            }),
+        );
+    }, [updatedScripts]);
+
```

Comment:
I think we should remove mocked items and stop passing it to TimelineView. We should read updatedScripts from store in ScriptsRow component. And in handlers like onDragEnd etc dispatch action to change scripts in store

Reply:
Because right now reorder scripts with dnd it will not get to store

Reply:
Looks like avatar mocks better to remove with avatar ticket

Reply:
Yes, agree
I will do it in this ticket https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/issues/165